CODEOWNERS

```
@@ -121,7 +121,7 @@ k8s/aat/common/sscs/sscs-tya-notif.yaml @hmcts/sscs
 k8s/**/common/sscs/sscs-bulk-scan.yaml @hmcts/sscs
 k8s/aat/common/sscs/sscs-cor-backend.yaml @hmcts/sscs
 k8s/aat/common/sscs/sscs-ccd-callback-orchestrator.yaml @hmcts/sscs
-k8s/aat/cluster-**/sscs/sscs-case-loader.yaml @hmcts/sscs
+k8s/**/cluster-**/sscs/sscs-case-loader.yaml @hmcts/sscs
```

I think you want this? but what you have might work
```suggestion
k8s/**/cluster-*/sscs/sscs-case-loader.yaml @hmcts/sscs
```